pennylane/__init__.py

```diff
@@ -15,35 +15,33 @@
 This is the top level module from which all basic functions and classes of
 PennyLane can be directly imported.
 """
-import pkg_resources
+import inspect
 
-from autograd import numpy
+import pkg_resources
 from autograd import grad as _grad
 from autograd import jacobian as _jacobian
-
-from semantic_version import Version, Spec
-
-import pennylane.operation
+from autograd import numpy
+from semantic_version import Spec, Version
 
 import pennylane.init
+import pennylane.operation
 import pennylane.templates
-from pennylane.templates import template, broadcast
 from pennylane.about import about
+from pennylane.templates import broadcast, template
 from pennylane.vqe import Hamiltonian, VQECost
 
-from .circuit_graph import CircuitGraph
-from .configuration import Configuration
 from ._device import Device, DeviceError
-from .collections import apply, map, sum, dot, QNodeCollection
 from ._qubit_device import QubitDevice
-from .measure import expval, var, sample, probs
+from ._version import __version__
+from .circuit_graph import CircuitGraph
+from .collections import QNodeCollection, apply, dot, map, sum
+from .configuration import Configuration
+from .io import *
+from .measure import expval, probs, sample, var
 from .ops import *
 from .optimize import *
-from .qnodes import qnode, QNode, QuantumFunctionError
+from .qnodes import QNode, QuantumFunctionError, qnode
```

Comment:
I'm assuming the above was because of isort? 😆

Reply:
Yes! :smile: